src/decryption/decryption.ts

```diff
@@ -16,17 +16,26 @@ interface IEncryptionOptions {
   ad: string;
 }
 
+/**
+ * @param passphrase Optional alias for [[key]]
+ */
 export async function decryptWithKeyDerivedFromString({
   serialized,
+  key,
   passphrase,
   encodingVersion = EncodingVersions.latest_version,
 }: {
   serialized: string;
-  passphrase: string;
+  key?: string;
+  passphrase?: string;
   encodingVersion?: EncodingVersions;
 }): Promise<Uint8Array | null> {
+  const phrase = key || passphrase;
```

Comment:
we only need to _deriveKeyWithOptions, if its **passphrase**, 
if it's already a **key** , then all we need is EncryptionKey class object out of it and don't need to _deriveKeyWithOptions 
https://github.com/Meeco/cryppo-js/blob/610f14507c2ea58916eff8212b85277f52b6f17a/src/encryption-key.ts